cypress/integration/clue/full/teacher_tests/teacher_workspace_spec.js

```diff
@@ -113,162 +120,165 @@ context.skip("Teacher Space", () => {
+                    rightNav.openSection('supports', 'teacher-supports');
+                    cy.waitForClueRightNavLoading()
+                    rightNav.getCanvasItemTitle('supports', 'teacher-supports').should('contain', title)
+                    rightNav.closeSection('supports', 'teacher-supports');
+                    rightNav.closeRightNavTab('supports')
+                })
+                it('switches to student view and checks for support badge', function () {
+                    cy.login('https://learn.concord.org', clueStudent)
+                    cy.wait(2000)
+                    cy.visit("https://learn.concord.org/users/" + clueStudent.studentUid + "/portal/offerings/" + offeringId + ".run_resource_html")
+                    cy.waitForSpinner();
+                    clueRightNav.getSupportBadge().should('exist').and('be.visible')
+                })
+                it('views published support, causing support badge to disappear', () => {
+                    rightNav.openRightNavTab('supports');
+                    rightNav.openSection('supports', 'teacher-supports');
+                    cy.waitForClueRightNavLoading()
+                    clueRightNav.getSupportBadge().should('not.exist').and('not.be.visible')
+                })
+                it('verify supports open in 2up view righthand workspace', () => {
+                    rightNav.openCanvasItem('supports', 'teacher-supports', title);
+                    clueCanvas.getRightSideInvestigationTitle().should('contain', title)
+                })
+                it('Logs back in as teacher, deletes support, and verifies confirmation dialog', function () {
+                    cy.login("https://learn.concord.org", clueTeacher)
+                    cy.wait(2000)
+                    cy.visit('https://learn.concord.org/portal/offerings/' + offeringId + '/external_report/25')
+                    cy.waitForSpinner();
+                    dashboard.switchView("Workspace")
+                    cy.wait(2000)
+                    rightNav.openRightNavTab('supports');
+                    rightNav.openSection('supports', 'teacher-supports');
+                    cy.waitForClueRightNavLoading()
+                    rightNav.deleteSupport(1) // Either give index or 'all'
+                    rightNav.confirmDeleteDialog()
+                })
+                it('switches back to student view verifies no supports', function () {
+                    cy.login('https://learn.concord.org', clueStudent)
+                    cy.wait(2000)
+                    cy.visit("https://learn.concord.org/users/" + clueStudent.studentUid + "/portal/offerings/" + offeringId + ".run_resource_html")
+                    cy.waitForSpinner();
+                    clueRightNav.getSupportBadge().should('not.exist')
+                    cy.visit("https://learn.concord.org/")
+                    cy.contains('Log Out').click({force:true})
+                })
+                after(function () {
```

Comment:
This seems like an odd use of the `after()` hook.

Reply:
This was an after statement from the code before I added my tests. I think it is supposed to log back into the teacher in order to undo a tile to return to a previous state of the teacher canvas before the teacher support was published. At least thats what it looks like to me.